src/form/input/typeahead/typeahead-input.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";

```

Comment:
I believe typeahead tests are not complete yet, right?

Reply:
Yes, sorry for the confusion